docs/GraFx-Studio/concepts/design-run/index.md

```diff
@@ -0,0 +1,63 @@
+# Design Mode and Run Mode
+
+GraFx Studio provides two modes for creating and testing Smart Templates: **Design Mode** and **Run Mode**. 
+
+These modes allow you to focus separately on designing the smart template and testing business logic.
+
+In **Design Mode**, you can freely arrange and style elements without triggering actions or interactions. This keeps the workspace clean, ideal for setting up and adjusting layouts, styles, and initial logic.
+
+Switching to **Run Mode** simulates the end-user experience (My Projects), showing how the template’s logic behaves live. In this mode, actions and visibility settings behave as they will in a real environment, letting you test how users will experience the template.
+
+## Design Mode
+
+**Design Mode** is optimized for layout and style tasks, with logic and actions inactive.
```

Comment:
It's only actions that are inactive in Design mode. Saying "logic and actions" sounds like there's more.

There's also a difference that in Design mode you can change the layout sizes. In Run mode, the layout sizes are read-only (you don't want to mess up the original design) but you can change the page size, which will resize the selected layout.